fedimint-bitcoind/src/bitcoincore_rpc.rs

```diff
@@ -152,34 +154,35 @@ where
 
     async fn get_block_height(&self) -> Result<u64> {
```

Comment:
you can use tracing #instrument instead of manual logging
```suggestion
    #[instrument(name = "BitcoindRpc::get_block_height", ret, err)]
    async fn get_block_height(&self) -> Result<u64> {
```

Reply:
Thanks for the recommendation. I have used the macro instead of manual logging.